doc/en/components/dock-manager.md

```diff
@@ -373,7 +373,61 @@ The shortcuts are as follows:
 
 Practice all of the above mentioned actions in the sample [`demo`](dock-manager.md#$Platform$-dock-manager-example).
 
-## Themes
+## Styling
+
+The Dock Manager uses a shadow DOM to encapsulate his styles and behaviors. As a result, you can't simply target its internal elements with the usual CSS selectors. That is why we expose components `parts` that can be targeted with the `::part` CSS selector.
+
+```css
+igc-dockmanager::part(content-pane) {
+  border-radius: 10px;
+}
+```
+
+In the following example, we demonstrate the ability of customizing the Dock Manager through some of the CSS parts that we've exposed.
+
+<code-view style="height: 600px"
+           data-demos-base-url="{environment:dvDemosBaseUrl}"
+           iframe-src="{environment:dvDemosBaseUrl}/layouts/dock-manager-styling"
+           alt="$Platform$ Dock Manager Styling Example"
+           github-src="layouts/dock-manager/styling">
```

Comment:
this sample is missing in Blazor repo, React repo, and Angular repo

https://github.com/IgniteUI/igniteui-blazor-examples/tree/vnext/samples/layouts/dock-manager

https://github.com/IgniteUI/igniteui-react-examples/tree/vnext/samples/layouts/dock-manager

https://github.com/IgniteUI/igniteui-angular-samples/tree/vNext/src/app/layouts/dock-manager/dock-manager-sample